src/client/driver/iframe-driver.js

```diff
@@ -70,6 +70,9 @@ export default class IframeDriver extends Driver {
 
                 if (inCommandExecution)
                     this._onReady(new DriverStatus({ isCommandResult: true }));
+            })
+            .then(() => {
+                // HACK: For IE9
```

Comment:
I believe we just need next tick here. Returning `delay(0)` in the previous promise should do a trick.